src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: block;
+  width: 200px;
+  height: 408px;
```

Comment:
Don't fix container size (if there is no such a requirement)
<img width="564" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/d67bb986-8e16-4fcb-bc10-d1946ac2bbe7">